common/setups/rasr/util/rasr.py

```diff
@@ -0,0 +1,204 @@
+__all__ = [
+    "RasrDataInput",
+    "RasrInitArgs",
+    "RescoreArgs",
+    "RasrSteps",
+]
+
+
+from collections import OrderedDict
+from typing import Dict, Optional
+
+import i6_core.meta as meta
+
+
+class RasrDataInput:
+    """
+    this class holds the data information for a rasr gmm setup:
+    - corpus object
+    - lexicon: file, normalization
+    - lm: file, scale, type
+    - concurrency
+    """
+
+    def __init__(
+        self,
+        corpus_object: meta.CorpusObject,
+        lexicon: dict,
+        lm: Optional[dict] = None,
+        concurrent: int = 10,
+    ):
+        """
+        :param corpus_object: corpus_file: Path, audio_dir: Path, audio_format: str, duration: float
+        :param lexicon: file: Path, normalize_pronunciation: bool
+        :param lm: filename: Path, type: str, scale: float
+        :param concurrent: concurrency for gmm hmm pipeline
+        """
+        self.corpus_object = corpus_object
+        self.lexicon = lexicon
+        self.lm = lm
+        self.concurrent = concurrent
+
+
+# when getting an zero weights error, set:
```

Comment:
Is this even the right place for this information?
Maybe make this part of a doc string.

Reply:
no it is not :)